SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -425,6 +427,36 @@ FHandoverChangeState USpatialActorChannel::CreateInitialHandoverChangeState(cons
 	return HandoverChanged;
 }
 
+void USpatialActorChannel::UpdateVisibleComponent(AActor* InActor)
+{
+	// Make sure that the Actor is not always relevant.
+	if (InActor->bAlwaysRelevant)
+	{
+		return;
```

Comment:
I wonder if we should call `NetDriver->RefreshActorVisibility(InActor, true);`
That would be based on if we are able to change `bAlwaysRelevant` at runtime or not. If we can change it after the actor has spawned, then it's possible to make a non-hidden actor, then mark it as hidden, then mark it as always relevant... Perhaps, worth a quick check.